src/App.js

```diff
@@ -1,26 +1,68 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
 
-function App() {
+import Board from './components/Board';
+
+const PLAYER_1 = 'x';
+const PLAYER_2 = 'o';
+
+const generateSquares = () => {
+  const squares = [];
+  let row = 0;
+  let col = 0;
+  let currentId = 0;
+  console.log(`row = ${ row }`);
+
+  while (row < 3) {
+    squares.push([]);
+    while (col < 3) {
+      squares[row].push({
+        id: currentId,
+        value: '',
+      });
+      col += 1;
+      currentId += 1;
+    }
+    row += 1;
+    col = 0;
+  }
+
+  return squares;
+}
+
+const App = () => {
+
+  const [squares, setSquares] = useState(generateSquares());
+
+  // Wave 1
+  // You will need to create a method to change the square 
+  //   When it is clicked on.
+  //   Then pass it into the squares as a callback
```

Comment:
After the README is in place, will the phrase "Then pass it into the squares as a callback" make sense? The thing that confuses me is the phrase "into the squares," is that talking about line 60?

Reply:
I'm with Dee that this feels a little out of place here.
Also this is about Wave 2 with the README update you're making [here](https://github.com/AdaGold/react-tic-tac-toe/pull/3), right?

Reply:
Can we move it to line 60?
Also, I see now that you update the wave number here. Thank you! lines 39 and 44 need updated wave numbers as well. (If only we could set them to auto-increment when a list insertion happens!)